### ISSUE TITLE
ImportError under Python 3.9

### ISSUE THREAD

**Issue by Zac-HD** [Instagram/LibCST#297]
Via https://github.com/Zac-HD/stdlib-property-tests/issues/12, attempting to `import libcst` fails on Python 3.9 with

```python-traceback
Traceback (most recent call last):
  File "t.py", line 3, in <module>
    import libcst
  File "libcst/__init__.py", line 187, in <module>
    from libcst._parser.entrypoints import parse_expression, parse_module, parse_statement
  File "libcst/_parser/entrypoints.py", line 25, in <module>
    _DEFAULT_PARTIAL_PARSER_CONFIG: PartialParserConfig = PartialParserConfig()
  File "<string>", line 8, in __init__
  File "libcst/_parser/types/config.py", line 132, in __post_init__
    raise ValueError(
ValueError: LibCST can only parse code using one of the following versions of
		    Python's grammar: 3.0, 3.1, 3.3, 3.5, 3.6, 3.7, 3.8. More versions may 
		    be supported by future releases.
```

Given that I'm using `libcst` in `hypothesmith` to test (among other things) pre-release Python versions, it would be lovely if libcst defaulted to the newest known grammar on newer-than-known Python versions rather than raising an error on import :smile:

**Comment by SimiCode** [comment]
I'm getting this error too when using MonkeyType
cc: @carljm

**Comment by SimiCode** [comment]
What can we do to make the change happen? 
Does it need testing? expansion?
@jimmylai

**Comment by Zac-HD** [comment]
Closed by #306.

(@thatch - the magic auto-close syntax only works in the PR description, not latter comments)
